keps/sig-network/3070-reserved-service-ip-range/README.md

```diff
@@ -0,0 +1,456 @@
+  - Components depending on the feature gate: kube-apiserver
+- [x] Other
+  - Describe the mechanism:
+  - The feature require to restart the apiserver to use the new allocation strategy for Services, however,
+  it will not require downtime in multi-control-plane environments.
+    
+###### Does enabling the feature change any default behavior?
+
+Dynamic allocated IPs will not be chosen from the first X values of the Service IP range, where X is obtained from the formula
+described in [Proposed Services ClusterIPs allocation model](#proposed-services-clusterips-allocation-model)
+This change is transparent to the user, since the IP is and was already random, it doesn't matter from which range the IP is obtained.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Nothing, this only affects new requests and how these new request obtain the dynamically assigned IPs.
+
+###### Are there any tests for feature enablement/disablement?
+
+No need to add new test, current tests should keep working as today.
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Current Services will not be affected, it doesn't matter in which range their IPs are allocated.
+This only affects the creation of new Services without an IP specified, there is no risk of impact on running workloads.
+
+###### What specific metrics should inform a rollback?
+
+The allocation logic already has the following metrics, that will be expanded with a new label containing information about the
+type of allocation requested (dynamic or static):
+
+  - allocated_ips
+  - available_ips
+  - allocation_total
+  - allocation_errors_total
+
+The increase of the errors metrics or the trend of the allocated_ips and available_ips doesn't change and new services are created or deleted.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+### Monitoring Requirements
+
+Current allocator metrics will be expanded with a new label to each metric containing information about the
+type of allocation requested (dynamic or static):
+
+  - allocated_ips
+  - available_ips
+  - allocation_total
+  - allocation_errors_total
+
+###### How can an operator determine if the feature is in use by workloads?
+
+A new metric will be added containing the information of the range configuration in the allocator.
+
+```go
+	allocatorInfo = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "kubernetes_allocator_info",
+			Help:           "A metric with a constant '1' value labeled by Service CIDR, ",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"cidr", "address_min", "address_max", "dynamic_range_offset"},
```

Comment:
address min and max doesn't add much value, having the cidr you know what is the first address

I think better to use 
```suggestion
		[]string{"cidr", "allocator_size", "dynamic_range_offset"},
```